docs/real-world-examples/ccfraud.md

```diff
@@ -31,6 +31,30 @@ In the next section, we will run a job in the AzureML workspace that will unpack
 
 Kaggle required username and a developer key, so we will first store safely those credentials in the workspace keyvault.
 
+### Using Azure CLI
+
+1. Get the signed-in user's object ID. 
```

Comment:
Can you add a context here?

Ex:

"In this section, we'll create an access policy for yourself, by using your AAD identifier (object id)"